app/models/job_application.rb

```diff
@@ -1,4 +1,4 @@
-class Apply < ApplicationRecord
+class Job_application < ApplicationRecord
```

Comment:
class name cannot have underscore. Change it.

Reply:
it has been changed